docs/component-guides/pipelines.md

```diff
@@ -0,0 +1,169 @@
+```
+
+Pass the cookie from your browser: 
+```bash
+# This is the "Domain" in your cookies. Eg: "localhost:8080" or "<ingress_alb_address>.elb.amazonaws.com"
+kubeflow_gateway_endpoint="<YOUR_KUBEFLOW_GATEWAY_ENDPOINT>"
+
+authservice_session_cookie="<YOUR_COOKIE>"
+
+namespace="<YOUR_NAMESPACE>"
+
+client = kfp.Client(host=f"http://{kubeflow_gateway_endpoint}/pipeline", cookies=f"authservice_session={authservice_session_cookie}")
+client.list_experiments(namespace=namespace)
+```
+
+If you want to set up application load balancing (ALB) with Dex, see [Exposing Kubeflow over Load Balancer](/docs/deployment/install/vanilla/vanilla/#exposing-kubeflow-over-load-balancer) and use the ALB address as the Kubeflow Endpoint.
+
+To do programmatic authentication with Dex, refer to the following comments under [issue #140](https://github.com/kubeflow/kfctl/issues/140) in the `kfctl` repository: [#140 (comment)](https://github.com/kubeflow/kfctl/issues/140#issuecomment-578837304) and [#140 (comment)](https://github.com/kubeflow/kfctl/issues/140#issuecomment-719894529).
+
+### **Cognito**
+
+```bash
+# This is the "Domain" in your cookies. eg: kubeflow.<platform.example.com>
+kubeflow_gateway_endpoint="<YOUR_KUBEFLOW_HTTPS_GATEWAY_ENDPOINT>"
+
+alb_session_cookie0="<YOUR_COOKIE_0>"
+alb_session_cookie1="<YOUR_COOKIE_1>"
+
+namespace="<YOUR_NAMESPACE>"
+
+client = kfp.Client(host=f"https://{kubeflow_gateway_endpoint}/pipeline", cookies=f"AWSELBAuthSessionCookie-0={alb_session_cookie0};AWSELBAuthSessionCookie-1={alb_session_cookie1}")
+client.list_experiments(namespace=namespace)
+```
+
+## S3 Access from Kubeflow Pipelines
+
+It is recommended to use AWS credentials to manage S3 access for Kubeflow Pipelines. [IAM Role for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) requires applications to use the latest AWS SDK to support the `assume-web-identity-role`. This requirement is in development, and progress can be tracked in the [open GitHub issue](https://github.com/kubeflow/pipelines/issues/3405).
+
+A Kubernetes Secret is required by Kubeflow Pipelines and applications to access S3. Be sure that the Kubernetes Secret has S3 read and write access.
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aws-secret
+  namespace: kubeflow
+type: Opaque
+data:
+  AWS_ACCESS_KEY_ID: <YOUR_BASE64_ACCESS_KEY>
+  AWS_SECRET_ACCESS_KEY: <YOUR_BASE64_SECRET_ACCESS>
+```
+
+- YOUR_BASE64_ACCESS_KEY: Base64 string of `AWS_ACCESS_KEY_ID`
+- YOUR_BASE64_SECRET_ACCESS: Base64 string of `AWS_SECRET_ACCESS_KEY`
+
+> Note: To get a Base64 string, run `echo -n $AWS_ACCESS_KEY_ID | base64`
+
+### Configure containers to use AWS credentials
+
+In order for `ml-pipeline-ui` to read these artifacts:
+
+1. Create a Kubernetes secret `aws-secret` in the `kubeflow` namespace.
+
+2. Update deployment `ml-pipeline-ui` to use AWS credential environment variables by running `kubectl edit deployment ml-pipeline-ui -n kubeflow`.
```

Comment:
As we did in this PR https://github.com/kubeflow/website/pull/3220

This section can be removed.